pkg/configtx/application.go

```diff
@@ -200,11 +199,9 @@ func (c *ConfigTx) ApplicationACLs() (map[string]string, error) {
 }
 
 // getACLs returns a map of ACLS for given config application.
+// Retrieval will panic if application group does not exist.
```

Comment:
nit: Probably makes more sense to move this comment up to `ApplicationACLs` since that's the user facing exported function

Reply:
Also not part of this PR, but any reason why the logic for getACLs isn't just in ApplicationACLs?

Reply:
I think because `ApplicationACLs()` only retrieves from the original  (unedited) config but `getACLs` retrieves from the provided config.

Reply:
ah I see we use `getACLs` for the removal